config.yaml

```diff
@@ -46,38 +46,7 @@ menu:
       url: https://github.com/go-gitea/
       weight: 60
       pre: github
-
```

Comment:
@lunny Commented code. I think it shouldn'd be there? If we ever need that again we can copy from website or docs repo.